spec/features/mi_bridges_driving_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.feature "MI Bridges Driving" do
+  scenario "successfully drives a SNAP Application", :js, :driving do
```

Comment:
![](http://reactiongifs.me/wp-content/uploads/2014/08/yeeeaaahhh-loki.gif)

Reply:
Might be worthwhile to add a comment here that tells you how to enable running this using the full chrome browser?

When it fails I imagine that's the first thing one would want to do is see this run on a browser. Thoughts?